lib/android-helpers.js

```diff
@@ -463,9 +463,10 @@ helpers.unlockWithUIAutomation = async function (driver, adb, unlockCapabilities
 };
 
 helpers.unlockWithHelperApp = async function (adb) {
-  logger.info("Unlocking screen");
-  await adb.forceStop(UNLOCK_HELPER_PKG_ID);
-  // then start the app twice, as once is flakey
+  await retryInterval(2, 2000, async () => {
```

Comment:
curly brackets are redundant in the lambda function body

Reply:
why should we retry force stop? I assume it is called `force`, because no retry is needed